lib/ansible/modules/database/mysql/mysql_query.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'supported_by': 'community',
+    'status': ['preview']
+}
+
+DOCUMENTATION = r'''
+---
+module: mysql_query
+short_description: Run MySQL queries
+description:
+- Runs arbitrary MySQL queries.
+- Pay attention, the module does not support check mode!
+  All queries will be executed in autocommit mode.
+version_added: '2.10'
+options:
+  query:
+    description:
+    - SQL query to run.
+    - Mutually exclusive with I(path_to_script).
+    type: str
+  positional_args:
+    description:
+    - List of values to be passed as positional arguments to the query.
+    - Mutually exclusive with I(named_args).
+    type: list
+  named_args:
+    description:
+    - Dictionary of key-value arguments to pass to the query.
+    - Mutually exclusive with I(positional_args).
+    type: dict
+  path_to_script:
```

Comment:
Also, the "query from file" feature overlaps the `mysql_db: { state: import }` feature, which means double codebase to maintain.

Maybe should we think of some way to make things simpler ?
@gundalow @felixfontein @Akasurde any good advice for this ?

Reply:
@bmalynovytch i thought about this, i don't mind to remove the option, thanks! (Would be happy to do it:)

Reply:
done :) good suggestion to remove the option at all! easy to maintain, fewer possible issues :)